apps/examples/testcase/tc_common.h

```diff
@@ -149,6 +155,29 @@ extern int total_fail;
 	} \
 }
 
+/*
+ * tc_performance : variadic macro for time measurements for function calls
+ *
+ * Args -
+ * fun : function
+ * sc_no : Number of arguments for function
+ * args : variable number of arguments to be passed to function
+ *
+ */
+#define tc_performance(fun, sc_no, args...) \
```

Comment:
I don't understand why you add this logic in tc. There is no inter-active with tc.

Reply:
Changed structure of code to match with structure for existing tc examples.